flashinfer/dllm/__init__.py

```
@@ -0,0 +1,37 @@
+    "BLOCK_EXTEND_V3_WITH_OFFSET_VARIANT_DECL",
+    # Cascade + block extend (SGLang style: causal + merge_state)
+    "block_extend_cascade",
+    "batch_block_extend_cascade",
+    "sglang_style_cascade_attention",
+    # Batch Prefill with offset versions
+    "BatchBlockExtendPagedOffsetWrapper",
+    "BatchBlockExtendRaggedOffsetWrapper",
+    # Batch Offset variant declarations
+    "_BATCH_BE_OFFSET_VARIANT_DECL",
+    "_BATCH_BE_OFFSET_VARIANT_DECL_FA3",
+]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)